authentik/stages/prompt/models.py

```diff
@@ -38,10 +39,17 @@ class FieldTypes(models.TextChoices):
 
     # Simple text field
     TEXT = "text", _("Text: Simple Text input")
+    # Long text field
+    TEXT_AREA = "text_area", _("Text area: Multiline Text Input.")
     # Simple text field
     TEXT_READ_ONLY = "text_read_only", _(
         "Text (read-only): Simple Text input, but cannot be edited."
     )
+    # Long text field
+    TEXT_AREA_READ_ONLY = "text_area_read_only", _(
```

Comment:
Long term it might make sense to have read-only as a toggle in the model rather than as a field type, but that's not really related to this PR

Reply:
Yup it seemed a bit repetitive to me, but I didn't want to introduce any unnecessary changes, especially to the UI.